packages/devtools_app/test/inspector/utils/inspector_tree.dart

```diff
@@ -12,13 +12,14 @@ import 'package:flutter_test/flutter_test.dart';
 InspectorTreeController inspectorTreeControllerFromNode(
   RemoteDiagnosticsNode node,
 ) {
-  final controller = InspectorTreeController()
-    ..config = InspectorTreeConfig(
-      summaryTree: false,
-      treeType: FlutterTreeType.widget,
-      onNodeAdded: (_, __) {},
-      onClientActiveChange: (_) {},
-    );
+  final controller =
+      InspectorTreeController(gaId: 'inspectorTreeControllerFromNode')
```

Comment:
what about making gaId optional and only making the calls into GA when gaId is not null?

Reply:
I initially avoided it since I would be changing to code to accomodate tests. 
Since there is a second voice that wondered it I am ok with doing it then :D